patches/tModLoader/Terraria/Localization/Content/fr-FR/tModLoader.json

```diff
@@ -3,22 +3,22 @@
 		// Main Menu
 		"PatreonSupport": "Considerez de nous supporter financièrement: ",
 		"MenuModBrowser": "Navigateur",
-		// "MenuDownloadMods": "Download Mods",
-		// "MenuDownloadModsDescription": "The in-game Mod Browser to find, download, and update your Mods.",
+		// "MenuDownloadMods": "Télécharger des Mods",
+		// "MenuDownloadModsDescription": "Le Bavigateur de Mods dans le jeu pour trouver, télécharger, et mettre à jour votre Mods.",
```

Comment:
Minor typo here, it's "Navigateur" not Bavigateur, a comma just before an "et" is usually discouraged and "votre" should be in plural ; "vos"